src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -61,10 +61,11 @@ export default class CheckoutPlugin extends Plugin {
             );
         }
 
-        const { locale, originKey, environment, paymentMethodsResponse } = adyenCheckoutConfiguration;
+        const { locale, originKey, clientKey, environment, paymentMethodsResponse } = adyenCheckoutConfiguration;
         const ADYEN_CHECKOUT_CONFIG = {
             locale,
             originKey,
+            clientKey,
```

Comment:
Can the component handle all the scenarios?

1. Origin key is provided, Client key is empty string
2. Origin key is provided, Client key provided (which one is preferred by the component?)
3. Origin key is empty string, Client key is provided

Maybe it's better to add the clientKey to the ADYEN_CHECKOUT_CONFIG if provided otherwise the origin key?

Reply:
Yes, I actually tested this and is looking good:

1. Origin key is provided, Client key is empty string: Origin key is used successfully
2. Origin key is provided, Client key provided (which one is preferred by the component?): [Client key is used successfully](https://github.com/Adyen/adyen-shopware6/pull/93/files#diff-94c3ec56b7a92d80e1953086cde504fc4919edaefee871ac714fdf3fd3e57562R214) 
3. Origin key is empty string, Client key is provided: Client key is used successfully.

Reply:
Nice thanks for the confirmation 👍